vendor/bat-native-ledger/src/bat/ledger/internal/credentials/credentials_common.cc

```diff
@@ -121,12 +79,13 @@ void CredentialsCommon::BlindedCredsSaved(
 
 void CredentialsCommon::GetSignedCredsFromResponse(
     const CredentialsTrigger& trigger,
-    const std::string& response,
+    const ledger::UrlResponse& response,
     ledger::ResultCallback callback) {
   base::Value parsed_response(base::Value::Type::DICTIONARY);
-  ParseSignedCredsResponse(response, &parsed_response);
+  const ledger::Result result =
```

Comment:
we receive `RETRY_SHORT`, but it's ignored

Reply:
This code receives `RETRY_SHORT` as a suggestion from a previous code review, but I guess I'm not sure why we wanted to make that logic change in the first place. None of this particular code has ever handled `RETRY_SHORT` before, from what I can see. It seems like `ParseSignedCreds` should return `RETRY` to remain consistent with how things were before.

Reply:
Spoke offline, I'll add a `RETRY_SHORT` handler to `GetSignedCredsFromResponse`.